### DISCUSSION TITLE
Wyscout V3 shot x and y

### DISCUSSION THREAD

**Discussion by fubininho** [PySport/kloppy#316] [Q&A] [answered]
#Wyscout V3 doesn't specify where the shot end in (x,y) terms, but it's possible to aproximate based on the zones of the goal they define.

<img width="1059" alt="wy_shot_def" src="https://github.com/PySport/kloppy/assets/48478393/cfd822d7-0eff-4acf-9ec8-9d1b53afb979">

Though it's not 100% accurate, when the shot is on target is possible to direct the y coordinate to me in the middle of the third zones. When it's off target, it's possible to set a bit more far away from the goal.

This means that:
- gl = (68/2)+(7.32/3) = 36,44
- gc = 34
- gr = (68/2)-(7.32/3) = 31,56
- ol ~= (68/2)+7.32 = 41,32
- or ~= (68/2)-7.32 = 26,68

Currently all shots are located to the corner, which IMHO is less accurate.

Is my thinking correct, or am I missing something? Is it worth it to make this change?

See: https://github.com/PySport/kloppy/blob/master/kloppy/infra/serializers/event/wyscout/deserializer_v3.py#L135

**Comment by probberechts** [comment, marked as answer]
In my opinion, it makes sense to make this change. We did already implement it for the V2 data deserializer in https://github.com/PySport/kloppy/commit/eebc7b0bdfae26956c59a7b1e9dc666322858639

**Reply by fubininho** [reply]
It's easy fix then, I'll convert to an issue and open the PR. Thanks.

**Comment by JanVanHaaren** [comment]
Great suggestion. This change would be very welcome!

**Comment by fubininho** [comment]
Addressed in #320
